app/schemas/consolidated-schema.js

```diff
@@ -279,6 +279,7 @@ const schema = {
       name: 'revenue2019',
       minLength: TEXT_MIN_LENGTH,
       maxLength: TEXT_MAX_LENGTH,
+      pattern: '^[0-9]+(.[0-9]{1,2})?$',
```

Comment:
For this pattern, I wonder if it might be useful to save it as something like CURRENCY_PATTERN similar to the min and max length you set up. Just in case we want to change it later to have commas or $ sign or something might make it easier. And for the pattern, we could maybe force the first digit to be [1-9], to make sure we don't get leading zeros

Reply:
Yeah good call, ill get the changes in.